nexuse2e-webapp/src/main/frontend/src/app/conversation-list/conversation-list.component.ts

```diff
@@ -39,12 +36,27 @@ export class ConversationListComponent implements OnInit {
         endDate: ConversationListComponent.END_DATE_DEFAULT,
       },
     },
+    {
+      fieldName: "conversationId",
+      filterType: FilterType.TEXT,
+    },
+    this.choreographyFilter,
+    this.participantFilter,
+    {
+      fieldName: "status",
+      filterType: FilterType.SELECT,
+      allowedValues: ["ERROR", "PROCESSING", "IDLE", "COMPLETED"],
+    },
   ];
   activeFilters: ActiveFilter[] = [];
 
-  constructor(private dataService: DataService) {}
+  constructor(private dataService: DataService) {
+  }
 
-  async ngOnInit() {}
+  async ngOnInit() {
+    this.participantFilter.allowedValues = await this.dataService.getParticipantIds();
+    this.choreographyFilter.allowedValues = await this.dataService.getChoreographyIds();
```

Comment:
Is it useful to get this values in parallel?
```suggestion
    [this.participantFilter.allowedValues, this.choreographyFilter.allowedValues] = Promise.all([this.dataService.getParticipantIds(), this.dataService.getChoreographyIds()]);
```

Reply:
Thank you!